### PR TITLE
[ISSUE-28] Add package specific documentation

### PR DESCRIPTION
**Pull request by lampajr**
**Thank you for submitting this pull request**

fix:
- https://github.com/kiegroup/chain-status/issues/28

**referenced Pull Requests**: 
- https://github.com/kiegroup/chain-status/pull/31
- https://github.com/kiegroup/chain-status/pull/34

Add package specific documentation:

- `webpage` documentation which includes specification and usage of its related `generate-app` action.
- `action` documentation which provides the js tool purpose and usage, either from command line or using `generate-data` action.

### CODE REVIEW

**Comment by lampajr** [comment]
I'll promote this as 'ready to review' once https://github.com/kiegroup/chain-status/pull/31 will be merged

File: packages/webpage/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Project Status Web Page
+
+This package contains the source code of React web application.
```
Comment (Ginxo) [inline packages/webpage/README.md:3]:
```suggestion
This package contains the source code of React web application. The web application has to be instantiated by your own configuration like information page markdown file. 
The data consumed by the app is preprocessed by [action package](https://github.com/kiegroup/chain-status/tree/main/packages/action) so no need for a backend and can be easily published on services like Github Packages.
```

File: packages/webpage/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+```
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.\
+Keep in mind that the page will reload if you make edits on the source code.
+
+4. Build the whole project [run it on the project's root]
+```bash
+$ yarn build
+```
+
+## Action Flow
```
Comment (Ginxo) [inline packages/webpage/README.md:33]:
```suggestion
## Installation
```

File: packages/action/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Generate Data JS Tool 
+
+This package provides a JS tool which basically consumes information from Github API or Jenkins API and generates a JSON file which is retrieved by the [React web application](https://github.com/kiegroup/chain-status/tree/main/packages/webpage/README.md).
```
Comment (Ginxo) [inline packages/action/README.md:3]:
```suggestion
This package provides a NodeJS CLI tool which basically consumes information from Github API and/or Jenkins API and generates a JSON file which is retrieved by the [React web application](https://github.com/kiegroup/chain-status/tree/main/packages/webpage/README.md).
```

File: packages/action/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Usage
+
+This tool is an easy to use JS tool, in order to use you just need to install all required libraries, build the project and run the automatically generated distribution file at `packages/action/dist/index.js`.
```
Comment (Ginxo) [inline packages/action/README.md:7]:
```suggestion
This tool can be easy use as a NodeJS CLI tool or as a Github Action

### NodeJS CLI Usage

In order to use you just need to install all required libraries, build the project and run the automatically generated distribution file at `packages/action/dist/index.js`.
```

File: packages/action/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+In order to locally run this, you simply need to run the [cli tool](./src/bin/cli.js) providing at least all required arguments (more details in [inputs](#inputs) section)
+
+Here an usage example:
+
+```bash
+$ node packages/action/src/bin/cli.js -t 'Title' -st 'Subtitle' --token <GH-TOKEN> -df https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml [-o <PATH-TO-REACT-WEBAPP-DATA>]
+```
+
+### Action Flow
```
Comment (Ginxo) [inline packages/action/README.md:44]:
```suggestion
### Github Action Flow
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

**Pull request merged**